Dockerfile

```diff
@@ -8,9 +8,8 @@
 # https://packagemanager.posit.co/cran/__linux__/jammy/2023-10-30
 
 # set proper base image
-ARG PLATFORM="linux/amd64"
 ARG R_VERS="4.3.1"
-FROM --platform=$PLATFORM rocker/r-ver:$R_VERS
+FROM rocker/r-ver:$R_VERS AS base
```

Comment:
out of curiosity, what does this do (`AS base`)?

Reply:
This names this stage, so that we can use if in the `FROM base` on L48. Then when we pass to buildx, we can tell it to not cache the `install-pacta` stage, but keep the cache for `base`

Reply:
ah, so it's like a label/tag for that layer 👍🏻